src/main/scala/com/github/agourlay/cornichon/http/HttpAssertions.scala

```diff
@@ -24,31 +28,49 @@ object HttpAssertions {
   case class HeadersAssertion(private val ordered: Boolean) extends CollectionAssertionSyntax[(String, String), String] {
     def is(expected: (String, String)*) = from_session_step[Iterable[String]](
       title = s"headers is ${displayTuples(expected)}",
-      key = LastResponseHeadersKey,
+      key = SessionKey(LastResponseHeadersKey),
       expected = s ⇒ expected.map { case (name, value) ⇒ s"$name$HeadersKeyValueDelim$value" },
       mapValue = (session, sessionHeaders) ⇒ sessionHeaders.split(",")
     )
 
     def hasSize(expected: Int) = from_session_step(
       title = s"headers size is '$expected'",
-      key = LastResponseHeadersKey,
+      key = SessionKey(LastResponseHeadersKey),
       expected = s ⇒ expected,
       mapValue = (session, sessionHeaders) ⇒ sessionHeaders.split(",").length
     )
 
     def contain(elements: (String, String)*) = {
       from_session_detail_step(
         title = s"headers contain ${displayTuples(elements)}",
-        key = LastResponseHeadersKey,
+        key = SessionKey(LastResponseHeadersKey),
         expected = s ⇒ true,
         mapValue = (session, sessionHeaders) ⇒ {
-        val sessionHeadersValue = sessionHeaders.split(InterHeadersValueDelim)
-        val predicate = elements.forall { case (name, value) ⇒ sessionHeadersValue.contains(s"$name$HeadersKeyValueDelim$value") }
-        (predicate, headersDoesNotContainError(displayTuples(elements), sessionHeaders))
-      }
+          val sessionHeadersValue = sessionHeaders.split(InterHeadersValueDelim)
+          val predicate = elements.forall { case (name, value) ⇒ sessionHeadersValue.contains(s"$name$HeadersKeyValueDelim$value") }
+          (predicate, headersDoesNotContainError(displayTuples(elements), sessionHeaders))
+        }
       )
     }
 
     override def inOrder: HeadersAssertion = copy(ordered = true)
   }
+
+  case class HttpListen(name: String, resolver: Resolver) {
+    def received_calls(count: Int) = AssertStep(
+      title = s"HTTP mock server '$name' received '$count' calls",
+      action = s ⇒ GenericAssertion(
+      expected = count,
+      actual = s.get(s"$name$nbReceivedCallsSuffix").toInt
+    )
+    )
+
+    def first_received_body = received_body_nb(1)
+
+    def received_body_nb(nb: Int) = {
```

Comment:
For my particular use case, I don't want to rely on the order of the webhooks I sent (because async).

I guess I could get `s"$name$receivedBodiesSuffix"` from the session myself, but it would be cool to have a helper function for that 👍